packages/manager/apps/pci-savings-plan/src/components/CreatePlanForm/CreatePlanForm.tsx

```diff
@@ -236,6 +246,11 @@ const CreatePlanForm: FC<CreatePlanFormProps> = ({
 
   const onChangeQuantity = (v: number) => setQuantity(v);
 
+  const handleQuantityChange = (event: OdsInputChangeEvent) => {
```

Comment:
should be wrapped in a useCallback

Also the onChangeQuantity function above doesn't do anything and could be removed